test/e2e/storage/utils/utils.go

```diff
@@ -383,9 +384,7 @@ func StartExternalProvisioner(ctx context.Context, c clientset.Interface, ns str
 					Name:  "nfs-provisioner",
 					Image: imageutils.GetE2EImage(imageutils.NFSProvisioner),
 					SecurityContext: &v1.SecurityContext{
-						Capabilities: &v1.Capabilities{
-							Add: []v1.Capability{"DAC_READ_SEARCH"},
-						},
```

Comment:
I'm surprised by this change... is this an example of the kind of workloads making `--seccomp-default` default to `true` would break on upgrade?

Reply:
Yes, at least with containerd v1.6.9 being used in CI. I tested the same in CRI-O and the test succeeded. I assume it's due to slight differences in the `RuntimeDefault` profile.